v22.2/ui-transactions-page.md

```diff
@@ -57,7 +57,7 @@ The **Statements** tab is selected. Click the **Transactions** tab and the **Act
 Active executions are polled every 10 seconds. Faster-running executions will potentially disappear upon each refresh.
 {{site.data.alerts.end}}
 
-The following screenshot shows the active statement execution for `SELECT city, id FROM vehicles WHERE city = 'paris'` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload):
+The following screenshot shows the active statement execution for `SELECT city, id FROM vehicles WHERE city = 'los angeles'` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload):
```

Comment:
The screenshot shows an `UPSERT` statement involving `boston`. Or am I not understanding what's happening?

Reply:
Thank you! I took so many screenshots and got them confused. I've committed the right ones now.